packages/server/lib/graphql/ServerActions.ts

```diff
@@ -105,17 +108,34 @@ export class ServerActions extends BaseActions {
     fs.writeFileSync(path.resolve(project.projectRoot, configFilename), code)
   }
 
-  async initializeOpenProject (args: LaunchArgs, options: OpenProjectLaunchOptions) {
-    await openProject.create(args.projectRoot, args, options)
+  async initializeOpenProject (args: LaunchArgs, options: OpenProjectLaunchOptions, browsers: FoundBrowser[]) {
+    await openProject.create(args.projectRoot, args, options, browsers)
+    if (args.testingType === 'e2e') {
+      this.ctx.activeProject!.setE2EPluginsInitialized(true)
+    }
+
+    if (args.testingType === 'component') {
+      this.ctx.activeProject!.setCtPluginsInitialized(true)
+    }
 
     return
   }
 
   async launchOpenProject (browser: BrowserContract, spec: any, options: LaunchOpts): Promise<void> {
+    debug('launching with browser %o', browser)
+
     return openProject.launch(browser, spec, options)
   }
 
   resolveOpenProjectConfig (): FullConfig | null {
     return openProject.getConfig() ?? null
   }
+
+  isFirstTime (projectRoot: string, testingType: Cypress.TestingType): boolean {
+    const config = JSON.parse(fs.readFileSync(path.join(projectRoot, 'cypress.json'), 'utf-8'))
```

Comment:
Prob need to `try/catch` this if the `cypress.json` doesn't exist, right?

Reply:
Yep, likely a lot of this code can go away/improve once the cypress.config.js changes are in, but I will add some error handling here for now